.github/workflows/main.yml

```diff
@@ -15,7 +15,7 @@ jobs:
           message: |
             After the CI passes:
 
-            - This branch can be previewed at [https://play.decentraland.zone/index.html?renderer=urn:decentraland:off-chain:renderer-artifacts:${{ github.head_ref }}](https://play.decentraland.zone/index.html?renderer=urn:decentraland:off-chain:renderer-artifacts:${{ github.head_ref }}&ENV=org)
+            - This branch can be previewed at https://play.decentraland.zone/?renderer-branch=${{ github.head_ref }}&ENV=org
```

Comment:
This won't break compatibility with PRs that are created by this week?

Reply:
play.zone already points to the new build. More info here: https://github.com/decentraland/explorer-technical-backlog/issues/23